src/test/java/org/cactoos/scalar/TernaryTest.java

```diff
@@ -133,11 +134,9 @@ void inputScalarValueConserved() throws Exception {
                 i -> new FormattedText("else: %d", i)
             ),
             new AllOf<>(
-                new IterableOf<>(
-                    new HasValue<>(new TextOf("1 equals 1")),
-                    new HasValue<>(new TextOf("else: 2")),
-                    new HasValue<>(new TextOf("else: 3"))
-                )
+                new HasValue<>(new TextOf("1 equals 1")),
+                new HasValue<>(new TextOf("else: 2")),
```

Comment:
@victornoel Wouldn't it be better as `new HasValue<>(new TextIs(...))`

Reply:
@andreoss why not, it's equivalent, but let's do it